ui/helpers/utils/notification.util.ts

```diff
@@ -301,8 +301,8 @@ export function getNetworkDetailsByChainId(chainId?: keyof typeof CHAIN_IDS): {
   nativeBlockExplorerUrl?: string;
 } {
   const fullNativeCurrencyName =
-    NETWORK_TO_NAME_MAP[chainId as keyof typeof NETWORK_TO_NAME_MAP];
-  const nativeCurrencyName = fullNativeCurrencyName.split(' ')[0];
+    NETWORK_TO_NAME_MAP[chainId as keyof typeof NETWORK_TO_NAME_MAP] ?? '';
+  const nativeCurrencyName = fullNativeCurrencyName.split(' ')[0] ?? '';
```

Comment:
Safeguards to prevent the `.split` issue.

I do want us to revisit this function in a future PR to make sure all other mappers are safe too.